packages/core/cms/sections.json

```diff
@@ -233,18 +767,39 @@
             "properties": {
               "key": {
                 "title": "Key",
-                "description": "Tip: For collections, use: productClusterIds",
+                "hint": "For collections, use: productClusterIds",
                 "type": "string",
                 "default": "productClusterIds"
               },
               "value": {
                 "title": "Value",
-                "description": "Tip: For collection 140, use: 140",
+                "hint": "For collection 140, use: 140",
                 "type": "string",
                 "default": "140"
```

Comment:
Where can wee see this magic number mapping? how can we know that `140` means collection? Would be good a way to see all available clusters.

Reply:
We can see in Collections page, I will improve the description